external_reports/bootstrap.yml

```diff
@@ -30,6 +30,20 @@ perl_distributions:
     affected:
       - perl_module_versions:
         distributed_library_version: '3.3.1'
+  - name: Kossy
+    last_version_checked: '0.60'
+    affected:
+      - perl_module_versions: '>=0.08,<=0.11'
+        distributed_library_version: '2.0.3'
+      - perl_module_versions: '>=0.12,<=0.16'
+        distributed_library_version: '2.1.0'
+      - perl_module_versions: '>=0.17,<=0.60'
+        distributed_library_version: '2.3.2'
```

Comment:
Was this meant for a different commit?

Reply:
Ah, no, but I didn't later change the name of the pull request when I tacked on the bootstrap stuff for Rossy.